docs/githubteacher.md

```diff
@@ -5,3 +5,5 @@
 **Title:**  Teacher
 
 **Objective:**  To learn more about workflow strategies
+
+**Favorite Vacation Spot:** Portland
```

Comment:
Which Portland are you talking about?  There are lots of Portlands in the :earth_africa: